tests/concurrency.rs

```diff
@@ -0,0 +1,28 @@
+//! Test that we can use an `FFT` object from multiple threads
+
+extern crate rustfft;
+
+use std::thread;
+
+use rustfft::FFTplanner;
+use rustfft::num_complex::Complex32;
+
+#[test]
+fn two_threads() {
+    let inverse = false;
+    let mut planner = FFTplanner::new(inverse);
+    let fft = planner.plan_fft(100);
+
+    let threads: Vec<thread::JoinHandle<_>> = (0..2).map(|_| {
+        let fft_copy = fft.clone();
+        thread::spawn(move || {
+            let mut signal = vec![Complex32::new(0.0, 0.0); 100];
+            let mut spectrum = vec![Complex32::new(0.0, 0.0); 100];
```

Comment:
One benefit of tests, especially integration tests like this, is that they give users an example of how to use the library. So a suggestion I have is, include the process of having data in the main thread that we send to each worker thread, compute the FFT, and send the result back to the main thread.

IT doesn't have to be the *best* way to do it, or even a recommended way, but it'll at least give users a starting point for figuring out their own solution